src/components/item/item-sliding.scss

```diff
@@ -17,26 +17,52 @@ ion-item-sliding .item {
 }
 
 ion-item-options {
-  @include position(0, 0, null, null);
-
+  // scss-lint:disable PropertySpelling
```

Comment:
@AmitMY so I should put it at two places? multi-dir and rtl mixin at the same time?